sw/device/tests/sim_dv/BUILD

```diff
@@ -1372,3 +1372,32 @@ opentitan_functest(
         "//sw/device/silicon_creator/lib/drivers:retention_sram",
     ],
 )
+
+opentitan_functest(
+    name = "ast_clk_rst_inputs",
+    srcs = ["ast_clk_rst_inputs.c"],
```

Comment:
It looks like `targets = ["dv"],` is missing here. 
Let me try that real quick (I will edit this commit directly).

Reply:
Indeed, it looks like this tag did the trick!